app/models/webhook.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+class Webhook < ApplicationRecord
+  include Authorizable
+
+  EVENT_POSTFIX = ".#{Foreman::Observable::DEFAULT_NAMESPACE}"
+
+  EVENT_ALLOWLIST = %w[
+    host_created host_updated host_destroyed
+    hostgroup_created hostgroup_updated hostgroup_destroyed
+    user_created user_updated user_destroyed
+    domain_created domain_updated domain_destroyed
+    subnet_created subnet_updated subnet_destroyed
+    build_entered build_exited status_changed
+  ].map { |e| e + EVENT_POSTFIX }.freeze
```

Comment:
What's this allow list for exactly?

Reply:
Wait isn't there a method which returns this list? Or why we need it here?

Anyway I agree this should be defined in core so other plugins can use it and modify it.

Reply:
I don't like filtering in the codebase and I tend to remove this completely, show them all but improve our UI so it's easier for users to pick items: https://community.theforeman.org/t/ux-dropdowns-with-many-items/20406

Again, let's keep it this way for now.